packages/block-library/src/image/edit.native.js

```diff
@@ -430,6 +436,50 @@ export class ImageEdit extends Component {
 		this.onSetSizeSlug( newValue );
 	}
 
+	onSetFeatured( mediaId ) {
+		const { closeSettingsBottomSheet } = this.props;
+		setFeaturedImage( mediaId );
+		closeSettingsBottomSheet();
+	}
+
+	getSetFeaturedButton( isFeaturedImage ) {
+		const { attributes, getStylesFromColorScheme } = this.props;
+
+		const setFeaturedButtonStyle = getStylesFromColorScheme(
+			styles.setFeaturedButton,
+			styles.setFeaturedButtonDark
+		);
+
+		const setFeaturedButton = () => (
+			<BottomSheet.Cell
+				label={ __( 'Remove as Featured Image ' ) }
```

Comment:
Just noticed, did we get the function names opposite? Looks like the current `setFeaturedButton()` shows a "Remove..." to the user 🤔

Reply:
Oh, wow, thanks for spotting that, not sure how that fell through the cracks. 🤦‍♀️ Mixup corrected in https://github.com/WordPress/gutenberg/pull/31705/commits/c309804e10855600f89fa12c473d7b27c2c95afa.